GOVERNANCE.rst

```diff
@@ -0,0 +1,139 @@
+SC members can call votes for decisions (technical or governance).
+
+The SC can veto decisions of the technical committee (TC) by voting in the SC.
+The TC can overwrite a veto with a 2/3rd majority vote in the TC.
+Decisions are documented in developer meeting notes and/or on the GitHub repository.
+
+The SC can change the governance structure, but only in a unanimous vote.
+
+Decision Process
+^^^^^^^^^^^^^^^^
+
+Decision of the SC usually happen in the developer meetings, via e-mail or public chat.
+
+Decisions are made in a non-confidential manner, by majority on the cast votes of SC members.
+Votes can be cast in asynchronous manner, e.g., over the time of 1-2 weeks.
+In tie situations, the chair of the SC acts as the tie breaker.
+
+Appointment Process
+^^^^^^^^^^^^^^^^^^^
+
+Appointed by current SC members in an unanimous vote.
+As a SC member, regularly attending and contributing to the weekly developer meetings is expected.
+
+SC members can resign or be removed by majority vote, e.g., due to inactivity, bad acting or other reasons.
+
+
+Technical Committee
+-------------------
+
+Current Roster
+^^^^^^^^^^^^^^
+
+- Ann Almgren
+- Marc Day
+- Candace Gilet
+- Kevin Gott
+- Axel Huebl
+- Andrew Myers
+- Weiqun Zhang
+- Michael Zingale
+
+See: `GitHub team <https://github.com/orgs/AMReX-Codes/teams/amrex>`__
+
+Role
+^^^^
+
+The technical committee (TC) is the core governance body, where under normal operations most ideas are discussed and decisions are made.
+Individual TC members can approve and merge code changes.
+Usually, they seek approval by another maintainer for their own changes, too.
+TC members lead - and weigh in on - technical discussions and, if needed, can call for a vote between TC members for a technical decision.
+TC members merge/close PRs and issues, and moderate (including block/mute) bad actors.
+The TC can propose governance changes to the SC.
+
+
+Decision Process
+^^^^^^^^^^^^^^^^
+
+Discussion in the TC usually happens in the developer meetings. Developer meetings can be scheduled by any member of the SC or TC.
+
+If someone calls for a vote to make a decision: majority based on the cast votes; we need 50% of the committee participating to vote. In the absence of a quorum, the SC will decide according to its voting rules.
+
+Votes are cast in a non-confidential manner.
+Decisions are documented in the developer meeting notes and/or on the GitHub repository.
+
+TC members can individually appoint new contributors, unless a vote is called on an individual.
+
+Appointment Process
+^^^^^^^^^^^^^^^^^^^
+
+TC members are the maintainers of AMReX.
+As a TC member, regularly attending and contributing to the developer meetings is expected.
+
+One is appointed to the TC by the steering committee, in a unanimous vote, or by majority vote of the TC. The SC can veto appointments.
```

Comment:
"New TC members can be suggested by either SC or TC members.   Confirmation of a new TC member requires a majority of votes cast by either the SC or the TC.   The SC can veto any new appointment."